Sources/Core/Log/LogOutputStream.swift

```diff
@@ -35,7 +35,7 @@
                 guard let bytes = ptr.baseAddress?.assumingMemoryBound(to: UInt8.self) else {
                     return
                 }
-                output.write(bytes, maxLength: data.count)
+                _ = output.write(bytes, maxLength: data.count)
```

Comment:
There were compile warnings which point out that the result of `write` function is not used.